go.mod

```diff
@@ -115,6 +115,8 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.23.4
 )
 
+require github.com/pkg/errors v0.9.1
```

Comment:
Can we get rid of this package in favor of the Go-native error handling? We recently got rid of it in #9319, so I'd prefer if we can avoid introducing it again.